frontend/src/metabase/common/hooks/use-entity-list-query/use-entity-list-query.ts

```diff
@@ -52,9 +52,10 @@ export const useEntityListQuery = <TItem, TQuery>(
   const error = useSelector(state => getError(state, options));
 
   const dispatch = useDispatch();
-  useEffect(() => {
+  useDeepCompareEffect(() => {
```

Comment:
Alternatively I could write my own machinery for deep comparison, but we already use `react-use` so I thought it's appropriate to use a built-in function